pacman-nodejs-app/docs/pacman-nodejs-app-single-cluster.md

```diff
@@ -21,67 +21,105 @@ clusters create "kube-ref-app" --zone "us-central1-f" \
 Once the cluster has been created, you'll want to log into the cluster:
 
 ```
-gcloud container clusters get-credentials kube-ref-app
+gcloud container clusters \
+  --project "<YOUR_PROJECT_ID>" \
+  --zone "us-central1-f" \
+  get-credentials kube-ref-app
 ```
 
-## Creating the MongoDB Service
+## Create MongoDB Resources
```

Comment:
I've updated this process to use a StorageClass and PersistentVolumeClaim for consistency with the federated examples